src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -6018,6 +6020,8 @@ private Color GetInternalColor()
                 internalColor = new Color(OnColorChanged, 0, 0, 0, 0);
             }
             Object.InternalRetrievingPropertyVector4(SwigCPtr, View.Property.COLOR, internalColor.SwigCPtr);
+            internalColor.CopyValuesFromDali();
```

Comment:
must this kind of code `someColorTypeVariable.CopyValuesFromDali();` be added in all places where color values are set into dali side?
I just can see your changes only in Textxxx classes, so I am just asking...
is this kind of POC? so should this kind of fix be done in other places also?

Reply:
Please remain some comments for `CopyValuesFromDali` API usage by summary keywords.